src/main/java/com/jcabi/github/RtRepo.java

```diff
@@ -217,8 +218,31 @@ public Notifications notifications() {
     }
 
     @Override
+    @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
     public Iterable<Language> languages() {
-        return Collections.emptyList();
+        final RtJson rtJson = new RtJson(
```

Comment:
I would name this `json` and the other one `object`, see http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html